sse.proxy.jwt/chart/templates/service-sse-server.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       containers:
         - name: sse-server
           args: ["-v", "-p", "8001", "-i", "7001"]
-          image: "zilla-examples/sse-server:latest"
+          image: "ghcr.io/aklivity/extras-sse-server:sha-42ad67e"
```

Comment:
`imagePullPolicy: Never ` should be removed, as it's not going to work without it.